site/docs/usage/self-hosting.md

```
@@ -378,6 +389,45 @@ The server component is optional; you can run evals locally or in CI/CD without
+     apiBaseUrl: http://your-server:3000
+     appBaseUrl: http://your-server:3000
+   ```
+
+   Replace `your-server` with your actual server address (e.g., `192.168.1.100`, `promptfoo.internal.company.com`, etc.)
+
+4. Then run: `promptfoo eval` followed by `promptfoo share`
+
+:::tip What to Expect
+After running `promptfoo share`, you should see output like:
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)